addons/mail/static/src/components/composer/composer.scss

```diff
@@ -184,26 +189,14 @@
     }
 }
 
-.o_Composer_button {
-    color: gray('600');
+.o_Composer_button.o-composer-is-compact {
+    border-left: none;
 
-    &.o-open{
-        background-color: gray('200');
-    }
-
-    &.o-composer-is-compact {
-        border-left: none;
+    :last-child {
         border-radius: 0 3px 3px 0;
```

Comment:
could use the new radius variable from the other PR?

Reply:
The PR you're mentioning : https://github.com/odoo/odoo/pull/53410

Probably could use yeah, once this one is merged